src/main/java/uk/gov/pay/api/model/ValidCreatePaymentRequest.java

```diff
@@ -13,6 +14,7 @@
  * Note that this class does not actually do any validation itself:
  * do that yourself before constructing an instance of this object
  **/
+@ApiModel(value = "CreatePaymentRequest")
```

Comment:
what's the effect of putting this `@ApiModel` here?
why does the "agreement_id" in this class not appear in the swagger.json file but everything else ("amount", "description", "reference", "return_url" etc) does?

Reply:
`@ApiModel(value = "CreatePaymentRequest")` will create model with the same name `CreatePaymentRequest` in swagger instead of defaulting to `ValidCreatePaymentRequest`

`agreement_id` is excluded for now and we intend to include it when Direct Debit is ready for use
https://github.com/alphagov/pay-publicapi/blob/master/pom.xml#L433

Reply:
ah so it's excluded in the pom.xml.